apps/src/p5lab/P5LabVisualizationHeader.jsx

```diff
@@ -98,6 +103,18 @@ class P5LabVisualizationHeader extends React.Component {
                     : msg.animationMode()}
                 </button>
               )}
+              {allowAnimationMode &&
+                this.props.isBlockly &&
+                experiments.isEnabled('backgroundsTab') && (
```

Comment:
we've run into performance issues when checking an experiment every time a component is rendered (granted, this will only check if `allowAnimationMode && this.props.isBlockly`, but it's good to be aware of this). since an experiment generally won't change after the page is loaded, i usually recommend checking if an experiment is enabled in the constructor or `componentDidMount` and storing that value for later

Reply:
@madelynkasula I just looked into this a little bit. What I found is that the experiment is checked once on page load and one additional time after certain actions that cause us to render again. The only time I saw it being checked repeatedly was when resizing the left column, but I don't think that's a common action. Do you think this is okay to merge or should it be reworked?

Reply:
okay, it's probably fine to leave as-is. we'll just have to keep this in mind if we run into an issue!